gammapy/cube/simulate.py

```diff
@@ -176,3 +176,37 @@ def sample_background(self, dataset):
         table.rename_column("DEC_TRUE", "DEC")
 
         return EventList(table)
+
+    def sample_psf(self, psf_map, events):
+        """Sample psf map.
+
+        Parameters
+        ----------
+        psf_map : `PSFMap`
+            PSF map.
+        events : `EventList`
+            Event list.
+
+        Returns
+        -------
+        events : `EventList`
+            Event list with reconstructed position columns.
+        """
+
+        energy_unit = events.table["RA_TRUE"].unit
```

Comment:
This is an angular unit, not an energy unit, no?
Suggest to remove that line, and just will "deg" below.
That's what the spec requires: https://gamma-astro-data-formats.readthedocs.io/en/latest/events/events.html

Reply:
Yes, sorry. A mistake in bad copy and paste.
I've considered only 'deg'.